components/blocks/subNewsLetters.tsx

```diff
@@ -119,10 +119,10 @@ export const SubNewsLetters = ({
   return (
     <div className="flex flex-col items-center justify-center">
       <h2
-        className="mb-2 mt-6 text-center font-helvetica text-4xl font-medium text-sswBlack"
+        className="mb-2 mt-6 text-center font-helvetica text-4xl font-medium text-white"
         dangerouslySetInnerHTML={{ __html: headerText }}
       ></h2>
-      <div className="mb-5 text-center text-base text-sswBlack">
+      <div className="mb-5 text-center text-base text-white">
```

Comment:
<img width="608" alt="image" src="https://github.com/SSWConsulting/SSW.Website/assets/38869720/62f739c9-1d4b-4f79-a6c4-cd1baa5ec025">

We need to conditionally change the text color depending on the background color

Reply:
You can find this component here https://app-sswwebsite-9eb3-pr-860.azurewebsites.net/?liveStream=1

Reply:
**Done**🙌
![image](https://github.com/SSWConsulting/SSW.Website/assets/52522913/ab4025f4-8850-490a-8f02-2a403b78e8eb)